internal/fetchclient/fetchclient.go

```diff
@@ -172,22 +181,24 @@ func (c *Client) fetchMaven(ctx context.Context, group string, name string) (str
 	q.Set("q", fmt.Sprintf("g:%s+AND+a:%s", group, name))
 	unescapedQuery, err := url.QueryUnescape(q.Encode())
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 	targetURL.RawQuery = unescapedQuery
 	request, err := http.NewRequestWithContext(ctx, http.MethodGet, targetURL.String(), nil)
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 	response, err := c.httpClient.Do(request)
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 	defer response.Body.Close()
 
 	var data struct {
 		Response struct {
-			Docs []struct {
+			NumFound int `json:"numFound"`
+			Start    int `json:"start"`
```

Comment:
Added these for future use (if we want to paginate over all versions). Right now scanning the first 20 results works ok.